### PR TITLE
app-dicts/myspell-ru-20180627 version bump

### PR DESCRIPTION
**Pull request by kuraga**
Signed-off-by: Alexander Kurakin <kuraga333@mail.ru>
Closes: https://bugs.gentoo.org/667146

Based on https://github.com/gentoo/gentoo/pull/9982

### CODE REVIEW

File: app-dicts/myspell-ru/myspell-ru-20180627.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline app-dicts/myspell-ru/myspell-ru-20180627.ebuild]: Could you try with EAPI=7? The eclass should support it, but I havent tested so cant be 100 % sure.

Comment (perfect7gentleman) [reply app-dicts/myspell-ru/myspell-ru-20180627.ebuild]: > Could you try with EAPI=7? The eclass should support it, but I havent tested so cant be 100 % sure.

EAPI=7 is okay

**Comment by kuraga** [comment]
Ping

**Comment by perfect7gentleman** [comment]
pong

**Comment by juippis** [comment]
I was waiting for an answer if EAPI-7 works or not with this, if it does, you should **always** use the latest EAPI available. So could you test it, and update your commit accordingly?

**Comment by perfect7gentleman** [comment]
> I was waiting for an answer if EAPI-7 works or not with this, if it does, you should **always** use the latest EAPI available. So could you test it, and update your commit accordingly?

I replied 2 weeks ago that EAPI=7 works.

**Comment by juippis** [comment]
So @kuraga should then update his commit in this PR. 
Due to https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin devs can't modify these ebuilds while merging, so I dont see a reason to push something I need to immediately update afterwards by myself (with a revbump), when it can be fixed easily and fast before pushing.

**Comment by kuraga** [comment]
Sorry, I didn't know. Thanks! Done. :smile:

**Comment by juippis** [comment]
It's good, thank you both :)

**Pull request closed without merging**